firmware/EsaSpeedSwitch/EsaSpeedSwitch.ino

```diff
@@ -25,182 +25,228 @@
 #include <stdint.h>
 
 static bool tuned = false;
+static bool locked = false;
+
+uint32_t startMillis = UINT32_MAX;
+
 #define LED_BUILTIN 13
-#define THRESHOLD 0x80 // min: 0x2b max: 0xc5
+#define THRESHOLD 0x80 //(128) min: 0x2b (43) max: 0xc5 (197)
+#define THRESHOLD_LOW 0x40
+
+#define SPEED_HIGH 25 // km/h
```

Comment:
Could you please set the default high speed back to 30km/h as it was before? :)

Reply:
Sure, i just forgot to change it :)